js/src/panes/pane.fullscreen.js

```diff
@@ -48,6 +48,7 @@ Wu.Fullscreen = Wu.Evented.extend({
 
 		// close trigger		
 		Wu.DomEvent.on(this._closer, 'click', this.destroy, this);
+		window.addEventListener('popstate', this.destroy.bind(this), false);
```

Comment:
This can be written as `Wu.DomEvent.on(window, 'popstate', this.destroy, this)`.  The `DomEvent.on` wrapper takes care of some cross-browser issues, so it's preferable to use it.